ILRepack.Tests/Utils/BamlDocumentMatcher.cs

```diff
@@ -153,7 +153,23 @@ private bool Equals(IEnumerable expected, IEnumerable actual)
 
         private List<BamlRecord> GetRelevantRecords(BamlDocument document)
         {
-            return document.Where(node => !(node is LineNumberAndPositionRecord || node is LinePositionRecord)).ToList();
+            var list = document.Where(node => !(node is LineNumberAndPositionRecord || node is LinePositionRecord)).ToList();
+            return OrderAssemblyInfoRecordsByName(list);
+        }
+
+        private List<BamlRecord> OrderAssemblyInfoRecordsByName(List<BamlRecord> bamlRecords)
+        {
+            var assemblyRecords = bamlRecords
+                .OfType<AssemblyInfoRecord>()
```

Comment:
I check a bit the unit test failure so out of 19 records only than AssemblyInfoRecord are relevant. Good to know. :-)

Reply:
The test has failed sometimes on my laptop because BamlReader.ReadDocument returns AssemblyInfoRecords sorted in the different order: in one case the WindowsBase-assembly base goes first, in another case the PresentationCore. 
I don't why, but the logical way to fix it is to sort the AssemblyInfoRecords. 
I just put the sorted recors at the end of the list, because the method is used only to compare the record list.